docs/src/main/asciidoc/ghpages.sh

```diff
@@ -145,6 +154,17 @@ function copy_docs_for_current_version() {
             fi
         done
         COMMIT_CHANGES="yes"
+    elif [[ "${BETA_RELEASE_BRANCHES_VALUE}" == "feature/spring-cloud-azure-passwordless-connection" ]] ; then
```

Comment:
here we should check wether current branch is one of the BETA_RELEASE_BRANCHES_VALUE